scene/main/node.cpp

```diff
@@ -2775,6 +2786,7 @@ void Node::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_children"), &Node::_get_children);
 	ClassDB::bind_method(D_METHOD("get_child:Node", "idx"), &Node::get_child);
 	ClassDB::bind_method(D_METHOD("has_node", "path"), &Node::has_node);
+	ClassDB::bind_method(D_METHOD("remove_all_children:Node"), &Node::remove_all_children);
```

Comment:
`remove_all_children` doesn't return `Node`, please remove the `:Node` part, it's wrong.